packages/workers-shared/package.json

```diff
@@ -34,7 +34,10 @@
 		"clean": "rimraf dist",
 		"deploy": "pnpm run deploy:router-worker && pnpm run deploy:asset-worker",
 		"deploy:asset-worker": "CLOUDFLARE_API_TOKEN=$WORKERS_DEPLOY_AND_CONFIG_CLOUDFLARE_API_TOKEN pnpx wrangler versions upload -c asset-worker/wrangler.toml",
+		"deploy:asset-worker-staging": "CLOUDFLARE_API_TOKEN=$WORKERS_DEPLOY_AND_CONFIG_CLOUDFLARE_API_TOKEN pnpx wrangler deploy -c asset-worker/wrangler.toml -e staging",
```

Comment:
Did you consider using `version upload` commands instead?

Reply:
Yes but for staging we don't want that, we want staging to be well, staging - it should just be what is most recent. We don't want to be releasing these manually for each change. If things break, that's good - we'll be alerted and can fix before a prod release.